great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -2597,6 +2597,10 @@ def _sqlalchemy_map_condition_index(
         accessor_domain_kwargs,
     ) = metrics.get("unexpected_condition")
 
+    result_format = metric_value_kwargs["result_format"]
+    if not result_format.get("unexpected_index_column_names"):
+        return None
```

Comment:
Allows for fallback behavior (Will return `None` instead of giving an error)

Reply:
Nit - can we  use `if "unexpected_index_column_names" not in result_format`. I like reserving `get` for when we actually use the retrieved value

Reply:
good point :) done